src/huggingface_hub/hub_mixin.py

```diff
@@ -363,6 +330,8 @@ def _save_pretrained(self, save_directory):
         model_to_save = self.module if hasattr(self, "module") else self
         torch.save(model_to_save.state_dict(), path)
 
+        return [path]
```

Comment:
The parent class's `_save_pretrained` doesn't specify returning anything, so adding this here throws me off a bit. But it seems like a nice idea, especially if folks are going to be overriding this. Should we maybe document this a bit to explain what's going on?

Reply:
In the end we didn't need this return so best thing for now is to remove it.